extensions/amp-story/validator-amp-story.protoascii

```diff
@@ -933,6 +933,51 @@ tags: {  # <amp-story-page-attachment> with no href
     value: "light"
   }
 }
+tags: {  # Same code as `<amp-story-page-attachment> with href`
+  html_format: AMP
+  tag_name: "AMP-STORY-PAGE-OUTLINK"
+  spec_name: "amp-story-page-outlink"
+  mandatory_ancestor: "AMP-STORY-PAGE"
+  mandatory_last_child: true
+  attrs: {
+    name: "layout"
+    mandatory: true
+    value: "nodisplay"
+  }
+  attrs: {
+    name: "theme"
+    value: "custom"
+    value: "dark"
+    value: "light"
+  }
+  attrs: {
+    name: "cta-accent-color"
+  }
+  attrs: {
+    name: "cta-accent-element"
+    value: "background"
+    value: "text"
+  }
+  attrs: {
+    name: "href"
+    mandatory: true
+    value_url: {
+      protocol: "http"
+      protocol: "https"
+    }
+  }
+  attrs: {
+    name: "cta-image"
+    value_url: {
+      # protocol: "http"  # TODO(honeybaderdontcare): readd once transformers in prod
```

Comment:
@honeybadgerdontcare Do we have a tracking bug for this? (Ditto 2 lines down.)

Reply:
This was completed in cl/361937561, so the comment is outdated and can be removed from -attachement. It has not been done for -outlink.

Reply:
For amp-story-page-outlink this is being done in cl/372160779.